app/gui/src/components/TabView.vue

```diff
@@ -147,4 +164,30 @@ const onSignOut = () => {
   min-height: 0;
   display: flex;
 }
+
+.closeButton {
+  --button-padding: 0px;
+  --icon-width: 12px;
+  --icon-height: 12px;
+
+  &:hover {
+    background-color: rgba(239, 68, 68, 0.8);
+  }
+
+  &.onMacOs {
```

Comment:
Since this will not vary from one element to another, maybe we should specify it higher up the DOM?